plugin-server/src/cdp/utils.ts

```diff
@@ -473,56 +475,49 @@ interface HogFunctionFilterResult {
 /**
  * Shared utility to check if an event matches the filters of a HogFunction.
  * Used by both the HogExecutorService (for destinations) and HogTransformerService (for transformations).
- * @param hogFunction The function to check filters for
- * @param filters The function filters
- * @param filterGlobals The globals to use for filter evaluation
- * @param options Optional parameters for telemetry and behavior control
  */
-export function checkHogFunctionFilters(
-    hogFunction: HogFunctionType,
-    filters: HogFunctionType['filters'],
-    filterGlobals: HogFunctionFilterGlobals,
-    options?: {
-        telemetryMatcher?: (teamId: number) => boolean
-        eventUuid?: string
-    }
-): HogFunctionFilterResult {
+export function checkHogFunctionFilters(options: {
+    hogFunction: HogFunctionType
```

Comment:
If we end up with multiple args, its often better to just turn it into a typed object as its easier to read on the caller side